README.md

```diff
@@ -1,2 +1,79 @@
-# angelCapstone-project
-This is a work in progress of my capstone project, I will be updating with more information this file. Happy coding :)
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Capstone Project of Module 1.
+
+# ArtScope
+
+> This is the final project of the module 1 in Microverse our approach was put in practice all the skills we got through our first module, such as Set up linters, Git and the Github Flow, Flexbox and Grid, HTML/CSS and JS best practices to recreate a webpage from scratch using visual content/guidelines.
+
+## Built With
+
+- HTML
+- CSS
+- JS
+
+## Live Demo
+
+[Live Demo Link](https://angeluray.github.io/angelCapstone-project/)
+
+## Video Presentation
+
+[Capstone project video presentation](https://youtu.be/QDnpQke4bnI)
+
+
+## Getting Started
+
+To get a local copy up and running it, follow these simple steps.
+
+### Prerequisites
+
+- Internet Service.
+- Basic Knowledge about Git and Github.
+- VSCode.
+
+### Setup
+
+1. Create an empty folder in your computer to store the clone.
+
+2. Copy the following link and paste it into your VSCode or your terminal:
+
+> git@github.com:angeluray/angelCapstone-project.git
+
+3. Store it in the file you created before.
+
+4. Create a new branch for working and make changes to adapt it according to your requirements.
+
+### Install
+
+To install linters please open the ***.github/workflows*** then get into ***linters.yml*** and check the instructions and commands.
+
+### Usage
+
+For educational purpose only, use it as a guideline.
+
+## Authors
+
+👤 **Angel Uray**
+
+- GitHub: [@angeluray](https://github.com/angeluray)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Special thanks to the autor of the original design Cindy Shin, The Microverse team and my Coding partners through this module.
+- ***“There are three ways to ultimate success: The first way is to be kind. The second way is to be kind. The third way is to be kind.” —Mister Rogers***
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[OPTIONAL] It looks like you forgot to add the MIT license to your project.

Reply:
MIT license correctly linked with README fie.